notebooks/TextAnalytics - Amazon Book Reviews with Word2Vec.ipynb

```diff
@@ -158,7 +170,6 @@
    "source": [
     "from mmlspark.automl import FindBestModel\n",
     "bestModel = FindBestModel(evaluationMetric=\"AUC\", models=trainedModels).fit(ptest)\n",
-    "bestModel.getRocCurve().show()\n",
```

Comment:
Does this no longer work?

Reply:
sorry mark, this doesn't work in Test Environment, but works in EUAP, added this line.